daemon/ipam.go

```diff
@@ -292,6 +292,10 @@ func (d *Daemon) allocateIPs() error {
 			}
 		}
 	}
+	if option.Config.EnableNodePort {
+		log.Infof("  NodePort-Range: %d-%d", option.Config.NodePortMin, option.Config.NodePortMax)
```

Comment:
nit: This option (and all config options) is already printed to the log upon startup, though you would want to embed the default into the CLI argument declaration [1].

Reply:
Makes sense, I'll do that. Thanks for the hint below!